candidate_word.h

```diff
@@ -20,9 +20,7 @@
 #ifndef __CANDIDATE_WORD_H__
 #define __CANDIDATE_WORD_H__
 
-#include <QDeclarativeView>
-#include <QObject>
-#include <qdeclarative.h>
```

Comment:
Is there a reason or a link that explains why? I can fix it, but don't know why it's preferred.